### DISCUSSION TITLE
Color setup

### DISCUSSION THREAD

**Discussion by MarkyMarc35** [awawa-dev/HyperHDR#662] [Q&A]
Hi all,
My setup has one Raspberry pi 4, a usb grabber, 130 leds on ws2812b strip and my power supply.
The strip is split in 3 (I didn't put leds under the TV) with corners done with 90 degrees connectors.
Everything is powered from the start point.

I run a test patern on my tv to match the color of the leds to the color on screen and for the basic colors (red, green and blue) all 3 sections are showing the correct color but when it comes to magenta or even white, the last part of the strip is shows red or green.

It there a setting that i can use to correct this or could it be a hardware issue?

**Comment by awawa-dev** [comment]
Hi
Voltage drop on the last segment (that segment needs additional power injection: +5v&ground) or using not recommended LED direct control directly from RPi although usually it affects bigger setups. Or insufficient power supply or you have some connectivity issues because such 90 degree adapters could be problematic and limit the current flow (increased risk of voltage drop, temperature increase etc) or disturb the data signal.

**Reply by MarkyMarc35** [reply]
Thank you for your suggestions. 
My power supply is a 15A, should be enough. 
I will add a power injection on the last segment. 
If it doesn't work I will change the 90 degree adapter for a wired connector.
